htdocs/wp-content/themes/owc-formulieren/assets/js/editor/personal-heading/edit.js

```diff
@@ -0,0 +1,336 @@
+import { __ } from '@wordpress/i18n';
+import {
+	useBlockProps,
+	MediaUpload,
+	MediaUploadCheck,
+	RichText,
+	InspectorControls,
+} from '@wordpress/block-editor';
+import {
+	Card,
+	CardHeader,
+	CardBody,
+	CardDivider,
+	Button,
+	Panel,
+	PanelBody,
+	PanelRow,
+} from '@wordpress/components';
+
+const ALLOWED_MEDIA_TYPES = ['image'];
+
+const Edit = ( props ) => {
+	const blockProps = useBlockProps( {
+		className: 'openpdd-personal-heading',
+	} );
+
+	const { attributes, setAttributes } = props;
+	const {
+		textContent,
+		bgImageMorningId,
+		bgImageMorningUrl,
+		bgImageAfternoonId,
+		bgImageAfternoonUrl,
+		bgImageEveningId,
+		bgImageEveningUrl,
+	} = attributes;
+
+	const fallbackInstructions =
+		<p>{__( 'U heeft geen rechten om foto\'s aan te passen. Neem contact op met de beheerder.', 'owc-formulieren' )}</p>;
+
+	const onUpdateMorningImage = ( image ) => {
+		let imageUrl = image.sizes.large.url || image.url;
+
+		setAttributes( {
+			bgImageMorningId: image.id,
+			bgImageMorningUrl: imageUrl,
+		} );
+	};
+
+	const onRemoveMorningImage = () => {
+		setAttributes( {
+			bgImageMorningId: undefined,
+			bgImageMorningUrl: undefined,
+		} );
+	};
+
+	const onUpdateAfternoonImage = ( image ) => {
+		let imageUrl = image.sizes.large.url || image.url;
+
+		setAttributes( {
+			bgImageAfternoonId: image.id,
+			bgImageAfternoonUrl: imageUrl,
+		} );
+	};
+
+	const onRemoveAfternoonImage = () => {
+		setAttributes( {
+			bgImageAfternoonId: undefined,
+			bgImageAfternoonUrl: undefined,
+		} );
+	};
+
+	const onUpdateEveningImage = ( image ) => {
+		let imageUrl = image.sizes.large.url || image.url;
+
+		setAttributes( {
+			bgImageEveningId: image.id,
+			bgImageEveningUrl: imageUrl,
+		} );
+	};
+
+	const onRemoveEveningImage = () => {
+		setAttributes( {
+			bgImageEveningId: undefined,
+			bgImageEveningUrl: undefined,
+		} );
+	};
```

Comment:
Dit is allemaal erg duplicated code. Het is mooier om de value mee te geven als parameter. 

```
	const onUpdateImage = (timeOfDay, image) => {
		const imageUrl = image.sizes.large.url || image.url;

		setAttributes({
			[`bgImage${timeOfDay}Id`]: image.id,
			[`bgImage${timeOfDay}Url`]: imageUrl,
		});
	};

	const onRemoveImage = (timeOfDay) => {
		setAttributes({
			[`bgImage${timeOfDay}Id`]: undefined,
			[`bgImage${timeOfDay}Url`]: undefined,
		});
	};
```